sympy/integrals/manualintegrate.py

```diff
@@ -1319,10 +1320,14 @@ def eval_trig(func, arg, integrand, symbol):
         return sympy.sec(arg)
     elif func == 'csc*cot':
         return sympy.csc(arg)
+    elif func == 'sec':
+        return sympy.log(sympy.sec(arg) + sympy.tan(arg)) / arg.diff(symbol)
```

Comment:
This makes it different from what `sin`, `cos`, etc does and changes the meaning of the function.

Reply:
Earlier all the trigonometric function were defined in a way where the arguments were just `x` and not multiples of `x`. After making the changes all the tests passed